compiler/testData/codegen/bytecodeText/forLoop/forInRangeToCharConst.kt

```diff
@@ -9,5 +8,10 @@ fun test(): Int {
     return sum
 }
 
-// 0 IF_ICMPEQ
-// 1 IF_ICMPGT
\ No newline at end of file
+// 0 iterator
+// 0 getStart
+// 0 getEnd
+// 0 getFirst
+// 0 getLast
+// 0 getStep
+// 1 IF_ICMP
```

Comment:
// 1 IF_ICMPGE
// 1 IF

Reply:
I've tried to add it where it makes sense, however in some cases I couldn't be very specific with the instructions because of differences between IR and non-IR. One example is in `downTo` loops, where the condition for IR is something like `while (last <= inductionVar)` whereas for non-IR it is `while (inductionVar >= last)`, resulting in different `IF_ICMPXX` instructions.